_overviews/scala3-scaladoc/static-site.md

```diff
@@ -97,6 +97,19 @@ Predefined properties:
 - **hasFrame** when set to `false` page will not include default layout (navigation, breadcrumbs, etc.) but only token HTML wrapper to provide metadata and resources (js and css files). **This setting is not exported to the template engine.**
 - **layout** - predefined layout to use, see below. **This setting is not exported to the template engine.**
 
+You can also use redirection properties such as:
+
+- **redirectTo** - redirect to another page.
+
+- **redirectFrom** - redirect from another page.
+
+You can use them when you want to change the location of the page without breaking the links. Example:
+
+```
+---
+redirectTo: /docs/scala3-scaladoc/static-site.html
+---
+```
```

Comment:
`redirectFrom` is the most used one, so I think we should mention it before `redirectTo`.

`redirectTo` is used in an otherwise empty .md file, when we want to redirect to an external website (thus, it is impossible in such a case to use `redirectFrom`).

It would be good to explain a bit more how to use `redirectFrom`. Typically, when you move a page to a new location but want the old URL to still work, you add `redirectFrom: /absolute/path/to/old/url.html` in the frontmatter and Scaladoc generates an empty page at `/absolute/path/to/old/url.html` with a browser-based redirection to the new location. It would be good to also mention the Jekyll plugin [jekyll-redirect-from](https://github.com/jekyll/jekyll-redirect-from) that inspired the feature.

Reply:
Thank you @julienrf, it it better ?